front/src/components/HeaderMenu/MenuDesktop.tsx

```diff
@@ -18,12 +18,16 @@ import MenuIcon, { Icon } from "./MenuIcons";
 import { expandedMenuIndex } from "./expandedMenuIndex";
 import { useContext } from "react";
 import BaseSwitcher from "./BaseSwitcher";
+import { useBaseIdParam } from "hooks/useBaseIdParam";
 
 function MenuDesktop({ menuItemsGroups }: IHeaderMenuProps) {
   const { isOpen, onOpen, onClose } = useDisclosure();
   const { handleLogout } = useHandleLogout();
+  const { baseId: currentBaseId } = useBaseIdParam();
   const { globalPreferences } = useContext(GlobalPreferencesContext);
   const baseName = globalPreferences.selectedBase?.name;
+  const currentOrganisationHasMoreThanOneBaseAvailable =
+    (globalPreferences.availableBases?.filter((base) => base.id !== currentBaseId).length || 0) > 1;
```

Comment:
@fhenrich33 why do you filter the current base? For a 2-base organisation this results in `currentOrganisationHasMoreThanOneBaseAvailable` being false, and then the base switcher button is unclickable.

I think this is sufficient:    `(globalPreferences.availableBases?.length || 0) > 1`

Same in MenuMobile.tsx

Reply:
Whoops, missed a `= `for the `>= 1` condition. Thanks!

(Also this is a way to tell codewise that we don't want to show the current base as an option.)

Reply:
> Also this is a way to tell codewise that we don't want to show the current base as an option.

thx for the fix; I just thought that the options are constructed in the [BaseSwitchter](https://github.com/boxwise/boxtribute/pull/1742/files#diff-4f046224a7d284e2ab9daecc2f6be08e04b0bc9632e93a2ae613274fee0e35c4R25)